### PR TITLE
add debug info for kmeans

### PR DESCRIPTION
**Pull request by linnashuang**
add debug info to print out load and training time of kmeans workload

### CODE REVIEW

File: sparkbench/ml/src/main/scala/com/intel/sparkbench/ml/DenseKMeans.scala [hunk 1]
```diff
@@ -104,6 +105,8 @@ object DenseKMeans {
     val numExamples = examples.count()
 
     println(s"numExamples = $numExamples.")
+    println(s"load data time = ${System.currentTimeMillis() - cacheStart}")
```
Comment (winningsix) [inline sparkbench/ml/src/main/scala/com/intel/sparkbench/ml/DenseKMeans.scala:108]: Please add unit here. "ms"

File: sparkbench/ml/src/main/scala/com/intel/sparkbench/ml/DenseKMeans.scala [hunk 2]
```diff
@@ -119,6 +122,7 @@ object DenseKMeans {
 
     val cost = model.computeCost(examples)
 
+    println(s"training time = ${System.currentTimeMillis() - trainingStart}")
```
Comment (winningsix) [inline sparkbench/ml/src/main/scala/com/intel/sparkbench/ml/DenseKMeans.scala:125]: Please add unit here. "ms"

File: sparkbench/ml/src/main/scala/com/intel/sparkbench/ml/DenseKMeans.scala (hunk 1 above)
Comment (linnashuang) [reply sparkbench/ml/src/main/scala/com/intel/sparkbench/ml/DenseKMeans.scala]: fixed

File: sparkbench/ml/src/main/scala/com/intel/sparkbench/ml/DenseKMeans.scala (hunk 2 above)
Comment (linnashuang) [reply sparkbench/ml/src/main/scala/com/intel/sparkbench/ml/DenseKMeans.scala]: fixed

**Pull request merged**